docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -713,42 +745,29 @@ spec:
       key:
         key: tls.key
         secretName: collector
-  filters:
-  - name: my-parse
-    type: parse
-  - name: my-app-label  # <4>
-    type: openshiftLabels
-    openshiftLabels:
-      my-log-index: app
-  - name: my-infra-label  # <5>
-    type: openshiftLabels
-    openshiftLabels:
-      my-log-index: infra
   pipelines:
-  - name: my-app  # <6>
+  - name: my-app  # <4>
     inputRefs:
     - application
-    filterRefs:
-    - my-parse
-    - my-app-label
     outputRefs:
-    - es-output-by-label
-  - name: my-infra  # <7>
+    - es-app-output
+  - name: my-infra  # <5>
     inputRefs:
     - infrastructure
-    filterRefs:
-    - my-parse
-    - my-infra-label
     outputRefs:
-    - es-output-by-label
+    - es-infra-output
+  - name: my-audit  # <6>
+    inputRefs:
+    - audit
+    outputRefs:
+    - es-audit-output
 ----
 <1> service account `logcollector` must have the correct permissions (see Service Accounts above)
-<2> `es-output-by-label` is the output used in both pipelines
-<3> `index` is set to read the value from `.openshift.label.my-log-index` and prepend to the string "-write" or fallback to "app-write"
-<4> `my-app-label` filter is used to set the label "my-log-index=app" in the pipeline
-<5> `my-infra-label` filter is used to set the label "my-log-index=infra" in the pipeline
-<6> pipeline `my-app` includes application logs and labels them `app`
-<7> pipeline `my-infra` includes infrastructure logs and labels them `infra`
+<2> `es-app-output`, `es-infra-output` and `es-audit-output` are the outputs used in pipelines for route logs by log type
+<3> `index` must follow naming scheme `app-*`, `infra-*`  or `audit-*` (see: https://github.com/ViaQ/elasticsearch-openshift-ingest-plugin/blob/0db356e78766d678ca4d1e4ecb27d539fd29910b/src/main/java/org/elasticsearch/ingest/openshift/OpenshiftIndicesUtil.java#L60[ES DataModelPrefix code])
```

Comment:
We do not need to reference the code here.  I think the note below is sufficient